ref/prometheus.md

```diff
@@ -1,275 +0,0 @@
-Anbox Cloud gathers various performance metrics that you can access through API endpoints to create a monitoring solution. See [How to monitor Anbox Cloud](https://discourse.ubuntu.com/t/monitor-anbox-cloud/24338) for detailed information.
-
-The reference implementation for Prometheus (see [Example: Collect metrics](https://discourse.ubuntu.com/t/example-collect-metrics/17787)) provides some basic dashboards. You can, however, update them to fit your needs.
-
-The following sections list all metrics returned to Prometheus by the Anbox Cloud services that support Prometheus metrics.
```

Comment:
@morphis I feel the metrics that AMS provides can still be useful if users are building their own monitoring solution but since these metrics feel tied to prometheus, I removed the entire document. Let me know if we can still list them in a generic way.

Reply:
We need to keep this page for the sake of people who is going to build their own monitoring stack as you mentioned above.

Reply:
Reverted and brought back content related to prometheus metrics offered. Thanks!